### DISCUSSION TITLE
Default premissions for new users

### DISCUSSION THREAD

**Discussion by Khozhempo** [gogs/gogs#7014] [Q&A]
Hi!

Is any way to set default premissions for new users (in enable registration)?

Thank you.

**Comment by unknwon** [comment]
Hey, what kind of "default premissions" you're referring to?

**Reply by Khozhempo** [reply]
I mean that new (just registered) user can only keep comments.

**Reply by unknwon** [reply]
Got it, it is currently not supported, you can file a [feature request](https://github.com/gogs/gogs/issues/new?assignees=&labels=%F0%9F%8E%AF+feature&template=feature_request.yml) :)
